azure-pipelines.yml

```diff
@@ -1,8 +1,16 @@
 trigger:
 - main
+  paths:
+    exclude:
+    - subscriptions.json
```

Comment:
Are you sure this works❔ Might need to be
```suggestion
    - /Maestro/subscriptions.json
```

Reply:
Yup, looks like I was off by a directory